.github/workflows/conda-package.yml

```diff
@@ -8,50 +8,53 @@ env:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ubuntu-latest, windows-latest]
+        include:
+        - os: ubuntu-latest
+          pkgs_dirs: ~/.conda/pkgs
+          condarc: ~/.condarc
+          conda-bld: /usr/share/miniconda/conda-bld/linux-64/
+        - os: windows-latest
+          pkgs_dirs: C:\Users\runneradmin\.conda\pkgs
+          condarc: C:\Users\runneradmin\.condarc
+          conda-bld: C:\Miniconda\conda-bld\win-64\
         python: [3.8]
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-activate-base: true
+          activate-environment: ""
 
       - name: Set pkgs_dirs
         run: |
-          echo "pkgs_dirs: [~/.conda/pkgs]" >> ~/.condarc
+          echo "pkgs_dirs: [${{ matrix.pkgs_dirs }}]" >> ${{ matrix.condarc }}
       - name: Cache conda packages
         uses: actions/cache@v2
         env:
           CACHE_NUMBER: 0  # Increase to reset cache
         with:
-          path: ~/.conda/pkgs
+          path: ${{ matrix.pkgs_dirs }}
           key:
-            ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-python-${{ matrix.python }}-${{hashFiles('**/meta.yaml') }}
+            ${{ matrix.os }}-conda-${{ env.CACHE_NUMBER }}-python-${{ matrix.python }}-${{hashFiles('**/meta.yaml') }}
           restore-keys: |
-            ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-python-${{ matrix.python }}-
-            ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-
+            ${{ matrix.os }}-conda-${{ env.CACHE_NUMBER }}-python-${{ matrix.python }}-
+            ${{ matrix.os }}-conda-${{ env.CACHE_NUMBER }}-
 
-      - name: Add conda to system path
-        run: echo $CONDA/bin >> $GITHUB_PATH
       - name: Install conda-build
         run: conda install conda-build
       - name: Build conda package
-        run: |
-          CHANNELS="-c intel -c defaults --override-channels"
-          VERSIONS="--python ${{ matrix.python }}"
-          TEST="--no-test"
-
-          conda build \
-            $TEST \
-            $VERSIONS \
-            $CHANNELS \
-            conda-recipe
+        run: conda build --no-test --python ${{ matrix.python }} -c intel -c defaults --override-channels conda-recipe
       - name: Upload artifact
         uses: actions/upload-artifact@v2
         with:
-          name: ${{ env.PACKAGE_NAME }} ${{ runner.os }} Python ${{ matrix.python }}
-          path: /usr/share/miniconda/conda-bld/linux-64/${{ env.PACKAGE_NAME }}-*.tar.bz2
+          name: ${{ env.PACKAGE_NAME }} ${{ matrix.os }} Python ${{ matrix.python }}
```

Comment:
matrix.os = ubuntu-latest
runner.os = Linux

Test job uses runner.os for searching for artifacts.